app/assets/components/game-events/game-events.css.scss

```diff
@@ -0,0 +1,13 @@
+:host {
+  display: block;
+  margin-bottom: 20px;
+
+  paper-input-decorator {
+    display: inline-block !important;
+  }
+
+  paper-button {
+    font-size: 10px;
+    margin: 18px 0px 0px 0px;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `18px 0px 0px` instead of `18px 0px 0px 0px`<br>`0px` should be written without units as `0`